.evergreen/config.yml

```diff
@@ -415,10 +415,10 @@ axes:
   - id: "jruby"
     display_name: JRuby Version
     values:
-      - id: "jruby-9.2"
-        display_name: jruby-9.2
+      - id: "jruby-9.3"
```

Comment:
I woud test both 9.2 and 9.3 since it's the same effort as testing one version.

Reply:
Done